blog/2024/release-updates.adoc

```diff
@@ -0,0 +1,30 @@
+:title: Backwards compatibility in Keycloak releases
+:date: 2024-08-19
+:publish: false
+:author: Stian Thorgersen
+
+With four major releases of Keycloak every year it can be a daunting task to keep deployments up to date. Especially,
+since https://www.keycloak.org/docs/latest/upgrading/index.html#migration-changes[the number of breaking changes have drastically increased the last couple years].
+Combine this with the importance of patching deployments quickly for vulnerabilities, this can leave many deployments
+open to known vulnerabilities as the time and effort required to update to https://github.com/keycloak/keycloak/security/policy[the latest release] is to costly.
+
+Additionally, currently Keycloak client libraries are released together with the server, resulting in new major versions
+of a client library, where in fact there can be no changes at all, or perhaps only a bug fix or two.
+
+For these reasons, after Keycloak 26.0 is released there will be some changes to how Keycloak is being released:
+
+* Keycloak server will have 4 minor releases every year, and a major release every 2-3 years
+* Keycloak client libraries will be released separately
```

Comment:
This is pretty important change for users IMHO. Should we add some more rough details here on the support matrix of the client libs with Keycloak versions?

Reply:
Updating